### DISCUSSION TITLE
Improperly configured DNS

### DISCUSSION THREAD

**Discussion by eastbristolhops** [community/community#32641] [Other Feature Feedback, Questions, & Ideas]
Hello Everyone,

I have looked through a load of  other discussions but can't find an answer to this error so any help would be most apreciated

![image](https://user-images.githubusercontent.com/113283162/189522670-8bd35735-fee9-48cd-bb7d-6286f0807910.png)

I have setup the the DNS as follows with nameschap

![image](https://user-images.githubusercontent.com/113283162/189522716-4ed87fae-9e00-43c9-93b0-6a1bea69f59e.png)

 and the CNAME does seem to resolve if I click here https://toolbox.googleapps.com/apps/dig/#CNAME/www.eastbristolhops.co.uk

![image](https://user-images.githubusercontent.com/113283162/189522747-781fdf65-5a87-4830-a47c-4ac76b3046a4.png)

![image](https://user-images.githubusercontent.com/113283162/189522767-5166419b-b135-48cf-b45f-a112f174dbc8.png)

I have spoken to nameschaep and they have passed the buck and told me the problem is with github, what am I doing wrong? it has been under 24 hours since I set it up maybe 20 hours so I would have thought it would work by now.

**Comment by yoannchaudet** [comment]
Your DNS configuration looks good and I can see your certificate was issued https://eastbristolhops.co.uk.

DNS propagation can be slow at times (even with small TTL).
